conda.bzl

```diff
@@ -81,9 +81,17 @@ def _install_conda(rctx, installer):
         fail("Failure installing conda.\nstdout: {}\nstderr: {}".format(result.stdout, result.stderr))
     return "{}/condabin/conda{}".format(rctx.attr.conda_dir, CONDA_EXT_MAP[os])
 
+def _install_mamba(rctx, executable):
+    rctx.report_progress("Installing mamba")
+    mamba_with_version = "mamba={}".format(rctx.attr.mamba_version)
+    args = [rctx.path(executable), "install", "-n", "base", "-c", "conda-forge", mamba_with_version, "-y"]
```

Comment:
Now I'm also wondering if `-n base` is necessary here. Wouldn't it be the same without it?

Reply:
I tried first to do this without `-n base` and conda would hang, it would freeze up and the last thing it printed was:

```
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment:
```

With the spinning line.

I believe when you don't specify `-n base` it installs the package to the user's existing conda installation instead of the Bazel conda installation. Because I got this when I removed mamba:

```
gabrield@rmpcic-iml1:~/ws/iml/rules_conda/tests$ /var/tmp/gabrield_bazel_root/7d32730c8ec51482761d587a8c6da76b/external/conda/conda/condabin/conda remove mamba                                                  Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##
                                                                                                                                                                                                                   environment location: /home/gabrield/anaconda3

  removed specs:
    - mamba


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    xmltodict-0.12.0           |     pyhd3eb1b0_0          13 KB
    ------------------------------------------------------------
                                           Total:          13 KB

The following packages will be REMOVED:

  libsolv-0.7.16-h8b12597_0
  mamba-0.7.3-py38h9709c9f_0
  python_abi-3.8-2_cp38
  reproc-14.2.1-h36c2ea0_0
  reproc-cpp-14.2.1-h58526e2_0

The following packages will be DOWNGRADED:

  xmltodict                                     0.12.0-py_0 --> 0.12.0-pyhd3eb1b0_0


Proceed ([y]/n)? y


Downloading and Extracting Packages
xmltodict-0.12.0     | 13 KB     | ###################################################################################################################################################################### | 100% Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```

Note that it says the environment is at `~/anaconda3` instead of `/var/tmp...` where the Bazel conda is installed. I believe the `-n base` allows this rule to be hermetic.

Reply:
Oh, that's interesting. Please add some comment about this behavior to not forget about it.